custom_components/tuya_local/devices/gosund_wp9rtl_3_outlet_usb_child_lock_and_state.yaml

```diff
@@ -0,0 +1,166 @@
+name: Gosund 3-Outlet + USB Powerstrip with Child Lock and State Memory
+products:
+  - id: 000002uovo
```

Comment:
This id looks like the format for "model id". It should be product id here.

Reply:
So it is. I'm still learning the different things I can query on Tuya's developer platform. I hope I got the right thing this time. But it also looks like `WP9_RTL` is also technically the `product_name` -- is that ok as the value for `model`

Reply:
You can use Gosund for manufacturer and WP9_RTL as the model. If there is another model identifier on the packaging, make WP9_RTL the model_id, and use the more friendly name as model.

Reply:
I haven't seen the packaging in quite some time, so will leave as is for now unless there are other changes you'd like me to make.